api/controllers/project.js

```diff
@@ -281,10 +282,12 @@ exports.protectedGet = async (args, res) => {
 
   defaultLog.info("PIPELINE", commentPeriodPipeline);
 
+  defaultLog.info("the auth", args.swagger.params)
+
   try {
     var data = await Utils.runDataQuery('Project',
-      args.swagger.params.auth_payload.realm_access.roles,
-      args.swagger.params.auth_payload.sub,
+      args.swagger.params.auth_payload.client_roles,
```

Comment:
I feel like there should be a utility function getting this information, it would have saved you a lot of changes.  Also by having it in one place it prevents typos, not that i have seen any yet.